package.json

```diff
@@ -19,18 +19,25 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "axios": "^0.27.2",
+    "html-react-parser": "^3.0.8",
```

Comment:
For the future preview page. Reference will be somewhere at the bottom. 

ATM an api returns the html page to be sent as a string, but has a few errors :'(

Reply:
and edge cases I didn't anticipate but will test again.